components/Profile/ProfileForm.js

```diff
@@ -0,0 +1,65 @@
+import { StyledProfile, PictureProfile } from "./Profile.styled";
+import { ProfileFormStyled, ProfileFormInfoText } from "./Profile.styled";
+import {
+  FormSection,
+  FormInput,
+  FormLabel,
+} from "../EventForm/EventForm.styled";
+import { FormButtonWrapper } from "../EventForm/EventForm.styled";
+import Button from "../Button/Button";
+import { useModal } from "@/lib/useModal";
+
+export default function ProfileForm({
+  toggleEditMode,
+  userInfo,
+  handleSubmit,
+}) {
+  const { showModal } = useModal();
+  const { profilePicture, name, additionalInfo } = userInfo;
+  const editableInfo = [name, ...additionalInfo];
+
+  return (
+    <StyledProfile>
+      <PictureProfile
+        src={profilePicture.value}
+        alt={profilePicture.text}
+        height={128}
+        width={128}
+      />
+      <ProfileFormStyled
+        onSubmit={(event) => {
+          event.preventDefault();
+          showModal({
+            message: "Änderungen speichern?",
+            textButtonCancel: "Abbrechen",
+            textButtonConfirm: "Speichern",
+            onConfirm: () => handleSubmit(event),
+          });
+        }}
+      >
+        {editableInfo.map((info) => {
+          const { key, text, value, isRequired } = info;
+          return (
+            <FormSection key={key}>
+              <FormLabel htmlFor={key}>
+                {text} {isRequired && "*"}
+              </FormLabel>
+              <FormInput
+                required={key === "name"}
```

Comment:
I wonder why you don't use the isRequired flag directly here. But perhaps there is a reason for this. 🙃
```suggestion
                required={isRequired}
```

Reply:
Yesterday evening, after posting the PR I saw that and also wondered why I did not do it 🙃